src/modules/home/components/header/user-navigation.menu.tsx

```diff
@@ -34,30 +46,47 @@ export function UserNavigationMenu<T extends MenuProps>({ anchorEl, handleMenuCl
     setSelectedLanguage(language);
   };
 
+  async function handleLogout() {
+    try {
+      await api.get(AUTH_ENDPOINTS.logout);
+      queryClient.invalidateQueries({ queryKey: ['auth-user'] });
+      dispatch(removeUser());
+      navigate(ROUTES.root);
```

Comment:
No need to redirect users if the page is public.

Reply:
it's standard behavior at Airbnb website though